DOCS-1691: Adds instructions about missing logos for Lightspeed (#975)
MultiSafepay/docs: content/integrations/hosted/lightspeed_app/faq/missing-payment-method-logos.md

---
title : "How can I add missing payment method logos?"
meta_title: "How can I add missing payment method logos? - Can I upgrade to the new app? - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

> Payment method logos may be missing due to a theme that is causing them to not appear on your Lightspeed webiste. This can easily be fixed by following these instructions:

Reviewer: ```suggestion
Payment method logos may be missing due to a theme that is causing them to not appear on your Lightspeed website. This can easily be fixed by following these instructions:
```

1. Download the logo(s) from our [Github repo](https://github.com/MultiSafepay/MultiSafepay-icons)

> 2. Rename the file with caps lock formatting e.g applepay.png -> APPLEPAY.png

Reviewer: ```suggestion
2. Rename the file with upper case formatting e.g applepay.png -> APPLEPAY.png
```

> 3. Place the image file in _Design_ -> _Thema editor_ -> _Geavanceerd_ -> _Code bewerken_ -> _Assests_

Reviewer: ```suggestion
3. Place the image file in _Design_ -> _Thema editor_ -> _Geavanceerd_ -> _Code bewerken_ -> _Assests_
```

Does Lightspeed have an English language setting? If yes, please use the English labels

> _Please note that changes will not occur automatically and may take time to come into effect._

Reviewer: I believe the changes will occur naturally but take some time as the theme is not updated automatically. By stating that the changes do not occur automatically we suggest the user has to perform manual actions for the changes to occur.
